metrics/web/test/base/presentation/widgets/icon_label_button_test.dart

```diff
@@ -201,43 +255,36 @@ class _IconLabelButtonTestbed extends StatelessWidget {
   /// The padding around the [IconLabelButton] under test.
   final EdgeInsets contentPadding;
 
-  /// The icon of the [IconLabelButton] under test.
-  final Icon icon;
-
-  /// The padding around the [icon].
+  /// The padding around the icon.
   final EdgeInsets iconPadding;
 
-  /// The label of the [IconLabelButton] under test.
-  final String label;
+  /// The builder of an icon of the button under tests.
+  final HoverWidgetBuilder iconBuilder;
 
-  /// The [TextStyle] of the [label].
-  final TextStyle labelStyle;
+  /// The builder of a label of the button under tests.
+  final HoverWidgetBuilder labelBuilder;
 
   /// Creates the instance of this testbed.
   ///
   /// Both [iconPadding] and [contentPadding] defaults to [EdgeInsets.zero].
-  /// The [label] defaults to `label`.
-  /// The [icon] defaults to [Icons.add].
   const _IconLabelButtonTestbed({
-    this.label = "label",
-    this.icon = const Icon(Icons.add),
+    this.labelBuilder,
+    this.iconBuilder,
```

Comment:
Maybe we can set the default params for these fields to avoid passing them in each test?